### PR TITLE
AM-2750: Revert changes made in demo to ORM and Refresh Batch for testing

### PR DESCRIPTION
**Pull request by tomxelliott**

### CODE REVIEW

File: apps/am/am-org-role-mapping-service/demo-image-policy.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
     hmcts.github.com/prod-automated: disabled
 spec:
   filterTags:
-    pattern: '^pr-1429-[a-f0-9]+-(?P<ts>[0-9]+)'
+    pattern: '^prod-1429-[a-f0-9]+-(?P<ts>[0-9]+)'
```
Comment (mattnayler) [inline apps/am/am-org-role-mapping-service/demo-image-policy.yaml:9]: Is this pattern correct, i.e. it still contains PR number?

File: apps/am/am-role-assignment-refresh-batch/demo-image-policy.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ metadata:
     hmcts.github.com/prod-automated: disabled
 spec:
   filterTags:
-    pattern: '^pr-470-[a-f0-9]+-(?P<ts>[0-9]+)'
+    pattern: '^prod-470-[a-f0-9]+-(?P<ts>[0-9]+)'
```
Comment (mattnayler) [inline apps/am/am-role-assignment-refresh-batch/demo-image-policy.yaml:9]: Is this pattern correct, i.e. it still contains PR number?

**Review by mattnayler — CHANGES_REQUESTED** [review]

File: apps/am/am-role-assignment-refresh-batch/demo-image-policy.yaml (hunk 2 above)
Comment (tomxelliott) [reply apps/am/am-role-assignment-refresh-batch/demo-image-policy.yaml]: Have resolved error, well spotted

File: apps/am/am-org-role-mapping-service/demo-image-policy.yaml (hunk 1 above)
Comment (tomxelliott) [reply apps/am/am-org-role-mapping-service/demo-image-policy.yaml]: Have resolved error, well spotted

**Review by mattnayler — APPROVED** [review]

**Pull request merged**